application-tracking.md

```diff
@@ -0,0 +1,49 @@
+# Application Tracking
+
+## Principles
+
+### Why?
+
+We should not be asking why we should be tracking an event in an application, we should be tracking events by default. Events containing sensitive information such as user or card details should still be tracked but with the sensitive data removed or masked.
+
+### What?
+
+Applications should have tracking enabled for all features that a user can interact with. When new features are written adding tracking should be a requirement for the definition of done.
+
+In addition to the previous point extra explicit tracking should also be performed when additional information needs to be gathered. Examples of these are as follows:
+
+ * Adding an item to a basket - Product info required.
+ * Booking success - Product, upgrade and booking info required.
+ 
+### How?
+
+At an application level we should not be concerned with how data is tracked beyond calling our tracking module with the correct parameters, once this has been done we can trust that the tracking module will do its job. Currently we have more than one tracking system and have plans to change how these are implemented if at all. Abstracting the decision away from an application level allows changes to our current systems and implementing new systems to be much easier.
```

Comment:
`Currently we have more than one tracking system and have plans to change how these are implemented if at all.` does not read well. Try...

`Currently we have more than one tracking system and have plans to change how these are implemented. We also plan to deprecate some of tracking solutions.`